warzyw-templates/Mods/brawl/Mods/brawlHotaObjects/config/banks/churchyard.json

```diff
@@ -0,0 +1,102 @@
+{
+	"hota.mapObjects:core:creatureBank" :

```

Comment:
please change to "core:creatureBank"

Reply:
Did that, but either that's not the correct way either or I made a mistake somewhere else. Because the game still crashes when I click 'start game' in launcher. I tried with just these mods and al of their submods:
VCMI essential files
VCMI extras
Horn of the Abyss
warzyw-templates
English translation

Reply:
Nevermind. Problem solved. I just forgot to put directory 'content' in my mod structure. I knew I saw these symptoms before.